tests/test_utils.py

```diff
@@ -29,6 +29,7 @@
     "llama2_tune": "/tmp/test-artifacts/small-ckpt-tune-03082024.pt",
     "llama2_meta": "/tmp/test-artifacts/small-ckpt-meta-03082024.pt",
     "llama2_hf": "/tmp/test-artifacts/small-ckpt-hf-03082024.pt",
+    "llama2_reward_hf": "/tmp/test-artifacts/small-ckpt-hf-reward-12072024.pt",  # TODO (SalmanMohammadi)
```

Comment:
Sorry for being an American chauvinist but I changed the filename to `small-ckpt-hf-reward-07122024.pt` (really just want to make it consistent with the format of the other ones). Also I think you will need to update [cache_artifacts.sh](https://github.com/pytorch/torchtune/blob/78055fa7d1faa885df98d039c4d8059e3477cfc8/tests/cache_artifacts.sh#L15) correspondingly

Reply:
Oh how the Empire has fallen from grace.